_adr/87/index.adoc

```diff
@@ -1,48 +1,452 @@
+|kas_cluster_partition_limit
+|Gauge
+|Not available
+|instance_name
+|Maximum number of partitions per instance
+
+|kafka_namespace:haproxy_server_bytes_in_total:rate5m
+|kas_cluster_incoming_bytes_rate5m
+|Gauge
+|Not available
+|Not available (no label available to get the instance_name from)
+|Number of incoming bytes per second in the last five minutes per instance
+
+|kafka_namespace:haproxy_server_bytes_out_total:rate5m
+|kas_cluster_outgoing_bytes_rate5m
+|Gauge
+|Not available
+|Not available (no label available to get the instance_name from)
+|Number of outgoing bytes per second in the last five minutes per instance
+
+|kafka_topic:kafka_topic_partitions:count
+|kas_cluster_topic_total
+|Gauge
+|Not available
+|instance_name (source label: strimzi_io_cluster)
+|Total number of topics, excluding system topics, per instance
+
+|kafka_topic:kafka_topic_partitions:sum
+|kas_cluster_topic_partition_total
+|Gauge
+|Not available
+|instance_name (source label: strimzi_io_cluster)
+|Total number of topics across all topics, excluding system topics, per instance
+
+|consumergroup:kafka_consumergroup_members:count
+|kas_cluster_consumer_group_total
+|Gauge
+|Not available
+|instance_name (source label: strimzi_io_cluster)
+|Total number of consumer groups, excluding system consumer groups, per instance
+
+|kafka_namespace:kafka_server_socket_server_metrics_connection_count:sum
+|kas_cluster_connection_total
+|Gauge
+|Not available
+|instance_name
+|Total number of connections, excluding system connections apart from canary, per instance
+
+|kafka_namespace:kafka_server_socket_server_metrics_connection_creation_rate:sum
+|kas_cluster_connection_creation_rate
+|Gauge
+|Not available
+|instance_name
+|Rate of connections established per second, excluding system connections apart from canary, per instance
+
+|kafka_instance_spec_brokers_desired_count
+|kas_cluster_desired_broker_total
+|Gauge
+|Not available
+|instance_name
+|Number of desired brokers per instance
+
+|===
+
+#### Broker Metrics
+|===
+|Old Metric Name | Revised Metric Name | Type | Old Labels | Revised Labels | Description
+
+|kubelet_volume_stats_available_bytes
+|kas_broker_volume_available_bytes
+|Gauge
+|persistentvolumeclaim
+|volume (source label: persistentvolumeclaim)
+|Number of available disk space, in bytes, per broker
+
+|kubelet_volume_stats_used_bytes
+|kas_broker_volume_used_bytes
+|Gauge
+|persistentvolumeclaim
+|volume (source label: persistentvolumeclaim), (no label available to get the instance_name from but persistentvolumeclaim can be used as identifier)
+|Number of used disk space, in bytes, per broker
+
+|kafka_broker_quota_totalstorageusedbytes
+|kas_broker_storage_used_bytes
+|Gauge
+|broker_id, statefulset_kubernetes_io_pod_name, strimzi_io_cluster
+|broker_id, instance_name (source label: strimzi_io_cluster)
+|Number of storage used by partitions, in bytes, per broker
+
+|kafka_broker_quota_hardlimitbytes
+|kas_broker_storage_hard_limit_bytes
```

Comment:
we don't distinguish between hard and soft limit anymore.  suggest simply: `kas_broker_storage_limit_bytes`

Reply:
Are these two metrics the same then `kafka_broker_quota_hardlimitbytes` and `kafka_broker_quota_softlimitbytes`?

Reply:
Yes.   From point of view of the public API I would just expose 'limit bytes'.

Reply:
Thanks for confirming @k-wall, I've included `kafka_broker_quota_softlimitbytes` in the list of metrics to be removed.